dev-aws/kafka-shared-msk/customer-proposition/customer-proposition.tf

```diff
@@ -20,13 +20,13 @@ resource "kafka_topic" "uswitch_data_v1" {
 
 module "uswitch-data-projector" {
   source           = "../../../modules/tls-app"
-  produce_topics   = [kafka_topic.uswitch.data.v1]
+  produce_topics   = [kafka_topic.uswitch_data_v1.name]
```

Comment:
Does this need to be just `uswitch.data_v1`, following https://github.com/utilitywarehouse/kafka-cluster-config/pull/211?

Reply:
i thought it would need to match the structure of how the tf resource is defined in that PR, which is `uswitch_data_v1`?

Reply:
> i thought it would need to match the structure of how the tf resource is defined in that PR, which is `uswitch_data_v1`?

Sorry, typo on my end: I meant just `uswitch_data_v1` and not `uswitch_data_v1.name`